File: fitcore-backend/Entity/NutritionDay.cs
namespace fitcore_backend.Entity;

/// <summary>
/// Cводка питания за день
/// </summary>
public class NutritionDay
{
    public Guid Id { get; set; }
    public Guid UserId { get; set; }

    public DateTime Date { get; set; } = DateTime.UtcNow.Date;

    public double ConsumedCalories { get; set; }
    public double TargetCalories { get; set; }

    public User User { get; set; } = null!;
    public ICollection<Meal> Meals { get; set; }
}

File: fitcore-frontend/src/models/NutritionDay.ts
import {Meal} from "./Meal";

export interface NutritionDay {
    id: string;                // Идентификатор дня питания
    userId: string;            // Пользователь
    date: string;              // Дата
    consumedCalories: number;  // Сколько съедено калорий
    targetCalories: number;    // Цель по калориям
    meals: Meal[];             // Приемы пищи
}
